src/triton_gazebo/models/import_mesh/import_mesh.py

```diff
@@ -0,0 +1,67 @@
+import os
+import subprocess
+import sys
+import json
+import argparse
+
+
+REPO_ROOT = subprocess.check_output(["git", "rev-parse", "--show-toplevel"]).decode("utf-8")[:-1]
```

Comment:
This requires the script to be executed from somewhere in the repo, a better way would be to add the scripts dir(mentioned above) to the setup.py in triton_gazebo (exactly like how the launch dir is added), and use the get_package_share_directory function that the gazebo launch file uses so we can get the path to the convert mesh script from anywhere.